### ISSUE TITLE
will it work with blazor server 

### ISSUE THREAD

**Issue by kiranmaya** [miroiu/nodify#34]
hi ,
i read this https://docs.microsoft.com/en-us/aspnet/core/blazor/hybrid/tutorials/wpf?view=aspnetcore-6.0,
so will this work in blazor -server app ?

**Comment by miroiu** [comment]
Hi,
I don't see any reason why this would not work. Have you tried it?

**Comment by kiranmaya** [comment]
Nope, not yet. new to blazor,is there any kind of canvas for WPF rendering ?

**Comment by miroiu** [comment]
It's actually the other way around. You have a WPF app that displays razor pages inside web views (using BlazorWebView). 
https://learn.microsoft.com/en-us/aspnet/core/blazor/hybrid/tutorials/wpf?view=aspnetcore-6.0

I can't help too much here, so I'm gonna close this issue.
